build/dockerfiles/test_entrypoint.sh

```diff
@@ -0,0 +1,350 @@
+#!/bin/bash
+#
+# Copyright (c) 2018-2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+METAS_DIR=$(mktemp -d)
+
+function cleanup() {
+    rm -rf  "${METAS_DIR}";
+}
+trap cleanup EXIT
+
+RED="\e[31m"
+GREEN="\e[32m"
+RESETSTYLE="\e[0m"
+BOLD="\e[1m"
+
+function initTest() {
+    echo -e "${BOLD}\n🏃 ${1}${RESETSTYLE}"
+    rm -rf "${METAS_DIR:?}"/*
+    unset CHE_SIDECAR_CONTAINERS_REGISTRY_URL \
+          CHE_SIDECAR_CONTAINERS_REGISTRY_ORGANIZATION \
+          CHE_SIDECAR_CONTAINERS_REGISTRY_TAG
+
+}
+
+function assertFileContentEquals() {
+    file=$1
+    expected_metayaml=$2
+
+    if [[ $(cat "${file}") == "${expected_metayaml}" ]]; then
+        echo -e "${GREEN}✔${RESETSTYLE} Test passed!"
+    else
+        echo -e "${RED}✘${RESETSTYLE} Test failed!"
+        echo "Result:"
+        cat "${file}"
+        echo "Expected:"
+        echo "${expected_metayaml}"
+        exit 1
+    fi
+}
+echo -e "${BOLD}\n🏃🏃🏃 Running tests for entrypoint.sh: ${BASH_SOURCE[0]}${RESETSTYLE}"
```

Comment:
sometimes when you copy console code into editors (like JIRA) it breaks when trying to save the non-ascii code to the database. wondering what the value is here of a running icon repeated 3 times when the next word is also "Running" :)

Reply:
i would say it is a bug in JIRA. or screenshot. The goal of this emojis is to visually helps someone reading CI logs and quickly figure out what is going on and why it is failing if it fails.

Reply:
@nickboldt I made the emoji optional (could be overridden with env variable)

Reply:
cool, might help when using consoles on older terminal emulators. 

'twas just a nitpick (and yeah, it's a JIRA bug/sql limitation, I agree).